src/components/screens/dashboard/recentTransactions/recentTransactions.js

```diff
@@ -40,7 +40,7 @@ const RecentTransactions = ({ className, t, transactions }) => {
   const [isLoaded, setLoaded] = useState(!!transactions.data.length);
   const settings = useSelector(state => state.settings);
   const currentBlockHeight = useSelector(selectCurrentBlockHeight);
-  const activeToken = tokenMap[settings.token.active];
+  const activeToken = tokenMap[settings.token.active].key;
```

Comment:
`tokenMap[settings.token.active].key` is exactly the same as `settings.token.active`.